### PR TITLE
Implement SubjectViewer Translations

### PR DESCRIPTION
**Pull request by wgranger**
This PR sets up translations for the app and adds translations for the subject viewer. More translations will need to be added eventually for other content on the page. I decided to use `react-localize-redux` rather than `counterpart` because the former worked better with changing the language state and translations through Redux.

There are a few translations missing for the subject viewer which will need to be added eventually.

### CODE REVIEW

File: src/components/SelectedAnnotation.jsx [hunk 1]
```diff
@@ -203,7 +205,7 @@ class SelectedAnnotation extends React.Component {
     const notes = emptyText ? 'You cannot save an empty transcription.' : '';
     this.props.dispatch(togglePopup(
       <QuestionPrompt
-        confirm="Yes, delete"
+        confirm={this.props.translate('cribSheet.confirm')}
         deny="No, continue transcribing"
```
Comment (eatyourgreens) [inline src/components/SelectedAnnotation.jsx]: Should this be localised too?

File: package.json [hunk 2]
```diff
@@ -25,14 +25,15 @@
   "homepage": "https://github.com/zooniverse/scribes-of-the-cairo-geniza",
   "dependencies": {
     "classnames": "~2.2.5",
-    "counterpart": "^0.18.1",
+    "counterpart": "~0.18.6",
```
Comment (eatyourgreens) [inline package.json]: Is counterpart used anywhere? I couldn't see it.

File: src/components/ActiveCard.jsx [hunk 3]
```diff
@@ -26,18 +28,24 @@ class ActiveCard extends React.Component {
     };
   }
 
-  deactivateCard() {
-    this.props.dispatch(activateCard(null));
+  onDelete() {
```
Comment (eatyourgreens) [inline src/components/ActiveCard.jsx]: Have the new methods in this component crept in from a different PR? They don't seem to be related to translations.

Comment (eatyourgreens) [reply src/components/ActiveCard.jsx]: nvm. Are these existing methods that have been re-ordered by the linter?

File: src/components/SelectedAnnotation.jsx (hunk 1 above)
Comment (wgranger) [reply src/components/SelectedAnnotation.jsx]: We don't have a translation for that yet.

File: package.json (hunk 2 above)
Comment (wgranger) [reply package.json]: Yes, good call. I forgot to remove this after I decided on `react-localize-redux`.

File: src/components/ActiveCard.jsx (hunk 3 above)
Comment (wgranger) [reply src/components/ActiveCard.jsx]: Yep, it's kinda irritating when the linter complains about method orders, but I make the switch to remove the error.

File: src/components/SelectedAnnotation.jsx (hunk 1 above)
Comment (eatyourgreens) [reply src/components/SelectedAnnotation.jsx]: I'd localise this in the view here and put placeholder strings in the locale files. No big deal, though, if you're aware that it still needs to be done.

**Comment by eatyourgreens** [comment]
One minor comment about the "No, continue transcribing" label but I'm happy to merge this as-is.
